constraints/recursiveCatEvals.js

```diff
@@ -34,50 +34,49 @@ function isMaximal(parent, child){
 	else return true;
 }
 
-/* Function that takes a tree and the category of its root's parent node
-	and labels all the nodes in the tree as being minimal or maximal 
-	instance of whatever category k they are, where:
-	minimal = does not dominate any nodes of category k
-	maximal = is not dominated by any nodes of category k
-	and level ordering is assumed (a node of category level k 
-	will never be dominated by a node of category < k).
-	
-	Previous isMin or isMax labels are preserved.
-*/
 // Move this to the prosodic hierarchy file probably?
 var sCat = ["cp", "xp", "x0"];
 
-function markMinMax(mytree, parcat){
-	// Check for maximalitys
-	if(!mytree.hasOwnProperty('isMax')){
-		mytree.isMax = (mytree.cat !== parcat)
-	}
-	
-	// Check for minimality
-	if(!mytree.hasOwnProperty('isMin')){
-		mytree.isMin = isMinimal(mytree);
+/* Function that takes a tree and labels all the nodes in the tree as being
+ * a minimal or maximal instance of whatever category k they are, where:
+ * minimal = does not dominate any nodes of category k
+ * maximal = is not dominated by any nodes of category k
+ * and level ordering is assumed (a node of category level k will never be
```

Comment:
Usually what you've called "level ordering" is called "layering", but I think your definition here is good.

Reply:
whoops, looks like I confused terms from morphology-phonology interface and syntax-prosody interface. The latest commit says "layering"